app/src/main/java/eu/theinvaded/mastondroid/viewmodel/LoginViewModelContract.java

```diff
@@ -20,7 +20,7 @@
         String getClientSecret();
         String getClientId();
         void setUser(MastodonAccount account);
-        void showLoginError();
+        void showLoginError(Throwable loginError);
```

Comment:
I think View shouldn't care if error is Throwable or something else, it may be better to just pass string or 2 strings for title and message.